resources/js/components/Create.vue

```diff
@@ -1,44 +1,54 @@
-                        </div>
-                        <div class="form-group">
-                            <label class="w-full" for="body">Body:</label>
-                            <input type="text" class="form-control" id="body" placeholder="Enter body here" v-model="newTodoBody" @keyup.enter="addTodoBody">
-                        </div>
-                    </div>
-                </form>
             </div>
-            <div class="col-12">
-                <h1 class="p-2 text-2xl text-gray-800 font-semibold my-5">Posts:</h1>
-                <div v-for="todo in todos" :key="todo.id">
-                    <div class="col-6 float-lg-left ellipsis">
-                        <h2>Title:  {{todo.title}}</h2>
+            <form id="myForm" class="form">
+                <div class="p-2 w-full">
+                    <div class="form-group">
+                        <label class="w-full" for="title">Title:</label>
+                        <input type="text"  class="form-control" id="title" placeholder="Enter title here" v-model="newTodoTitle" @keyup.enter="addTodoBody">
                     </div>
-                    <div class="col-6 float-lg-right ellipsis">
-                        <h2>Body: {{todo.body}}</h2>
+                    <div class="form-group">
+                        <label class="w-full" for="body">Body:</label>
+                        <input type="text" class="form-control" id="body" placeholder="Enter body here" v-model="newTodoBody" @keyup.enter="addTodoBody">
                     </div>
-                    <router-link class="col-12 btn btn-lg btn-success" :to="{name:'Show', params: {id:todo.id}}" style="color: black">Read More</router-link>
+                </div>
+            </form>
+            <div class="col-12">
+                <div class="flex flex-wrap my-5 row">
+                    <h1 class="p-2 text-2xl text-gray-800 font-semibold my-5 col-12">Posts:</h1>
+                    <table class="table table-striped">
+                        <thead class="thead-dark">
+                        <tr>
+                            <th scope="col">Title</th>
+                            <th scope="col">Body</th>
+                            <th scope="col">Date/Time</th>
+                            <th scope="col"></th>
+                        </tr>
+                        </thead>
+                        <tbody v-for="(todo, index) in todos" :key="todo.id">
+                        <tr>
+                            <td>{{todo.title}}</td>
+                            <td>{{todo.body}}</td>
+                            <td></td>
+                            <td><router-link class="btn btn-success" :to="{name:'Show', params: {id:todo.id}}" style="color: black">Read More</router-link>
+                                <span class="btn btn-danger" @click="remove(index)">Delete</span></td>
```

Comment:
I think it'll be more robust to remove based on the todo ID rather than the index